packages/react-composites/src/composites/CallComposite/ScreenShare.tsx

```diff
@@ -84,8 +84,9 @@ export const ScreenShare = (props: ScreenShareProps): JSX.Element => {
 
     return (
       <VideoTile
-        displayName={screenShareParticipant?.displayName}
-        isMuted={screenShareParticipant?.isMuted}
+        displayName={
+          screenShareParticipant?.displayName ? `${screenShareParticipant?.displayName}'s screen` : undefined
```

Comment:
This will break localization. "'s screen" is not a valid suffix beyond English.

Reply:
I followed Figma's naming
![image](https://user-images.githubusercontent.com/82416644/131753429-867e547f-2b3a-4e31-9f89-465f55d9140d.png)